.linkspector.yml

```
@@ -0,0 +1,15 @@
+dirs:
+  - CHANGELOG
+  - docs
+ignorePatterns:
+  - pattern: "^http://localhost"
+  - pattern: "https://azure.microsoft.com/en-us/products/kubernetes-service"
+httpHeaders:
+  - url:
+      - "https://docs.github.com/"
+    headers:
+      Accept-Encoding: "zstd, br, gzip, deflate"
+aliveStatusCodes:
+  - 200
+  - 206
+  - 429
```

```suggestion
```

Status code 429 - Too Many Requests shouldn't be considered "alive". 
As shown in `.markdownlinkcheck.json` (with `"retryOn429": true`), it just retries the request.

While having such a retry feature is great, if it isn't available, I think it should return an error.